css/colors.css

```diff
@@ -0,0 +1,6 @@
+:root {
+  --black-one: #000000;
+  --white-one: #d8d8d8;
+  --dark-one: #25222e;
+  --gray-two: #4a4658;
```

Comment:
silly question: is this `--gray-two` just to match what we have in other repos? it feels funny to me to have `gray-two` without a `gray-one` (or even a plain old `gray`), so just seeing if I'm missing some context

Reply:
Mistake, that should be `gray-one`.

That said, I debated if we should name these the same as the website repo, but figured it would just be too strange to have `dark-three` and `white-two` floating around without antecedents.